src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyle = {};
+
+  if (sourceString.trim() === '') {
+    return {};
+  }
+
+  const arrFromString = sourceString.trim().split(';');
```

Comment:
Instead of using 'split' and then 'map' to create an array of objects, you could chain these methods for better readability. Remember to maintain the same indentation level for chained methods.